CMakeLists.txt

```diff
@@ -178,6 +180,15 @@ else()
   set(FILESYSTEM "")
 endif()
 
+option(VELOX_ENABLE_DUCKDB "Build duckDB to enable differential testing." ON)
```

Comment:
could we move line 20 here as well, for organization?

Reply:
> could we move line 20 here as well, for organization?

done